src/app/app.component.ts

```diff
@@ -321,6 +318,11 @@ export class AppComponent implements OnInit {
       this._updateBitCloutExchangeRate();
     }, 5 * 60 * 1000);
 
+    // Clear the avatar map every 5 minutes
```

Comment:
Can we just require a reload to get fresh avatars? I am worried this will spam the servers with requests every 5 mins